pkg/providers/aws/provider_postgres.go

```diff
@@ -379,6 +379,14 @@ func (p *PostgresProvider) reconcileRDSInstance(ctx context.Context, cr *v1alpha
 			Database: *foundInstance.DBName,
 			Port:     int(*foundInstance.Endpoint.Port),
 		}
+
+		if !annotations.Has(cr, ResourceIdentifierAnnotation) {
```

Comment:
I believe this path isn't covered by unit tests. Would you be able to add a test case for it, if possible?

Reply:
@cecobask Thank you for comment and discussion. This line is covered, but error is not (L 386). Problem actually seems not to cover the error, but in checking Annotations. It's Not done in any place for Redis, Blobstorage, Postgres. If we just mock this, as was discussed, we will do almost nothing, checking error message, but will add significant duplication of code in test (as to come to Annotation related code - we will need duplicate (for negative case) 2 tests: "test rds CreateReplicationGroup is called (valid cluster bundle subnets)" and "test rds exists and is available (valid cluster bundle subnets)". But this duplication in test is not look nice. Need see how to test annotations better in All places. For this probably will need do changes in Instance creation (in test) and in buildTestPostgresCR. Maybe you want open separate Jira? Thanks again for this comment.